dashboard/app/models/authentication_option.rb

```diff
@@ -29,12 +29,32 @@ class AuthenticationOption < ApplicationRecord
   before_save :normalize_email, :hash_email, :remove_student_cleartext_email,
     :fill_authentication_id
 
+  OAUTH_CREDENTIAL_TYPES = [
+    CLEVER = 'clever',
+    FACEBOOK = 'facebook',
+    GOOGLE = 'google_oauth2',
+    POWERSCHOOL = 'powerschool',
+    QWIKCAMPS = 'lti_lti_prod_kids.qwikcamps.com',
```

Comment:
should this string be `lti_prod_kids` instead of `lti_lti_prod_kids`?

Reply:
now looking at your other PR (#23275), i'm pretty sure the `lti_lti` is intentional, so feel free to ignore